sunpy/net/dataretriever/tests/test_eve.py

```diff
@@ -25,11 +26,11 @@
      ),
     (TimeRange('2012/5/5', '2012/5/6'),
      'http://lasp.colorado.edu/eve/data_access/evewebdata/quicklook/L0CS/SpWx/2012/20120505_EVE_L0CS_DIODES_1m.txt',
-     'http://lasp.colorado.edu/eve/data_access/evewebdata/quicklook/L0CS/SpWx/2012/20120506_EVE_L0CS_DIODES_1m.txt'
+     'http://lasp.colorado.edu/eve/data_access/evewebdata/quicklook/L0CS/SpWx/2012/20120506_EVE_L0CS_DIODES_1m.txt',
      ),
     (TimeRange('2012/7/7', '2012/7/14'),
      'http://lasp.colorado.edu/eve/data_access/evewebdata/quicklook/L0CS/SpWx/2012/20120707_EVE_L0CS_DIODES_1m.txt',
-     'http://lasp.colorado.edu/eve/data_access/evewebdata/quicklook/L0CS/SpWx/2012/20120714_EVE_L0CS_DIODES_1m.txt'
+     'http://lasp.colorado.edu/eve/data_access/evewebdata/quicklook/L0CS/SpWx/2012/20120714_EVE_L0CS_DIODES_1m.txt',
```

Comment:
The extra commas I've added here probably need removing.

Reply:
Commas at the end of a container don't add anything. They are though a good practice for version control if you think that list, tuple, ... will grow over time, so there's not 2 lines changed, but only 1 new addition each time. (Probably not the case here in the inner tuple, but would do on the outer list).